### PR TITLE
add ci checks for the ocm-4.4.0 branch of hive

### PR DESCRIPTION
**Pull request by staebler**

### CODE REVIEW

**Comment by dgoodwin** [comment]
prow-config failure looks like something may be wrong, IIRC there was a command you need to run in this repo after making changes? Almost looks like it's expecting the v1 branch config to have been removed?

**Comment by staebler** [comment]
> prow-config failure looks like something may be wrong, IIRC there was a command you need to run in this repo after making changes? Almost looks like it's expecting the v1 branch config to have been removed?

The prow-config passed. https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/8166/pull-ci-openshift-release-master-config/20314

**Comment by staebler** [comment]
Fixed clusters used for images, integration, unit, and verify jobs.

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -325,6 +325,17 @@ config_updater:
         ci/api-build01-ci-devcluster-openshift-com:6443:
         - ci
       name: ci-operator-misc-configs
+    ci-operator/config/**/*-hive-ocm-*.yaml:
+      clusters:
+        app.ci:
+        - ci
+        - ci-stg
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:332]: `ci-stg` should not be here anymore

File: core-services/prow/02_config/_plugins.yaml [hunk 2]
```diff
@@ -325,6 +325,17 @@ config_updater:
...
+        ci/api-build01-ci-devcluster-openshift-com:6443:
+        - ci
+        default:
+        - ci
+        - ci-stg
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:337]: ditto

File: core-services/prow/02_config/_plugins.yaml (hunk 1 above)
Comment (staebler) [reply core-services/prow/02_config/_plugins.yaml:332]: Fixed.

File: core-services/prow/02_config/_plugins.yaml (hunk 2 above)
Comment (staebler) [reply core-services/prow/02_config/_plugins.yaml:337]: Fixed.

**Comment by petr-muller** [comment]
Please run `make jobs` to regenerate the jobs, the way how jobs are generated changed in the meantime.

**Comment by staebler** [comment]
> Please run `make jobs` to regenerate the jobs, the way how jobs are generated changed in the meantime.

Done.

**Pull request merged**